### PR TITLE
decaffeinate: add page

### PR DESCRIPTION
**Pull request by idleberg**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/decaffeinate.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# decaffeinate
+
+> Move your CoffeeScript source to JavaScript using modern syntax.
```
Comment (sbrl) [inline pages/common/decaffeinate.md]: Perhaps `Convert your`.....`to modern JavaScript.` would read better?

File: pages/common/decaffeinate.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Convert a CoffeeScript file to JavaScript:
+
+`decaffeinate {{filename}}.coffee`
+
+- Convert a CoffeeScript v2 file to JavaScript:
+
+`decaffeinate --use-cs2 {{filename}}.coffee`
```
Comment (sbrl) [inline pages/common/decaffeinate.md]: Could this be `{{path/to/filename}}` please?

Also the same for the other examples.

File: pages/common/decaffeinate.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Convert require and module.exports to import and export:
+
+`decaffeinate --use-js-modules {{filename}}.coffee`
+
+- Convert a CoffeeScript, allow named exports:
```
Comment (sbrl) [inline pages/common/decaffeinate.md]: Perhaps ....`, allowing named exports` would read better?

**Comment by agnivade** [comment]
Sorry, you misunderstood me. I meant the .coffee should be within tokens if they are not compulsory. Let me update.

**Pull request merged**